.env.template

```diff
@@ -5,6 +5,7 @@ JITSI_PRIVATE_MODE=false
 JITSI_ISS=
 SECRET_JITSI_KEY=
 ADMIN_API_TOKEN=123
+SERVER_ID=
```

Comment:
You should really add some comments. Like:

```suggestion
# The server id is used to uniquely identify a cluster of WorkAdventure servers
# It will generate a "/server/json" file at the root of the front container.
# It is only used in rare circumstances. You can safely keep it empty.
SERVER_ID=
```